src/NaluWind.cpp

```diff
@@ -83,6 +83,12 @@ void NaluWind::advance_timestep()
         realm->advance_time_step();
 }
 
+void NaluWind::nonlinear_iteration(const int n)
```

Comment:
I think this is what you meant earlier, there is an inconsistency in the naming (nonlinear_iteration vs nonlinear_iterations). Also, in the future maybe we should add a for loop here? And have two parameters? One that controls frequency of exchange solution between codes (now fixed at 1) and another that runs a fixed number of picards (like what we have with the input nonlinear_its)

Reply:
we should make it consistent!

Reply:
Ok it's consistent ;) 

Do we want to add the second piece of Mike's comments now? Or wait?